File: 2st/Yeonwoo/주사위 게임1.py
def solution(a, b):
    answer = 0
    if a%2==1 and b%2==1:
        answer += (a**2) + (b**2)
    elif a%2==1 or b%2==1:
        answer += 2*(a+b)
    elif a%2==0 and b%2==0:
        answer += abs(a-b)
    return answer

File: 4st/Seohyun/programmer/주사위 게임 1.java
public class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        if(a%2==1 && b%2==1)    //홀수
            answer=(a*a)+(b*b);
        else if(a%2==1 && b%2==0 || a%2==0 && b%2==1)
            answer=2*(a+b);
        else if(a%2==0 && b%2==0){
            answer=Math.abs(a-b);
        }
        return answer;
    }
} 